src/runtime_src/core/pcie/driver/linux/xocl/subdev/mb_scheduler.c

```diff
@@ -614,6 +631,23 @@ cmd_abort(struct xocl_cmd *xcmd)
 	SCHED_DEBUGF("xcmd(%lu) [-> abort]\n", xcmd->uid);
 }
 
+static inline bool
+can_enable_timestamps(struct xocl_cmd *xcmd)
```

Comment:
Prefix with cmd_ for consistency; if this was c++ it would be a member function of a command class and I have tried to use that convention here for all object types.

Reply:
Got it. Fixed now.